frontend/src/components/Uploader/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Dropzone from 'react-dropzone'; //eslint-disable-line no-unused-vars
+import './style.css';
+
+export default (props) => (
+  <div className="well">
+    <Dropzone onDrop={props.onFileDropped} >
+      Drop files here or click to select.
```

Comment:
if we extend this into further development. I think we should restrict to just pictures?